src/sqlfluff/core/linter.py

```diff
@@ -46,6 +51,8 @@
 # Instantiate the linter logger
 linter_logger: logging.Logger = logging.getLogger("sqlfluff.linter")
 
+tblib.pickling_support.install()
+
```

Comment:
Could you add a comment to explain this and why it's here?

Reply:
Will do!

Reply:
Does it make sense for this to go in __init__.py? I feel it does

Reply:
I'll move it. I don't have a strong feeling about where, but I do think code like this feels a little weird anywhere. Gotta have it, though! Argh.